tests/integration/client/courseInstance/viewselector.test.tsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { waitForElement } from '@testing-library/react';
+import { render } from 'test-utils';
+import CoursesPage from 'client/components/pages/Courses/CoursesPage';
+
+describe('View selection dropdown', function () {
+  let coursePage: ReturnType<typeof render>;
+
+  beforeEach(function () {
+    coursePage = render(<CoursesPage />);
+  });
+  it('defaults to the default view', async function () {
+    return waitForElement(
+      () => coursePage.findByDisplayValue(/Default/, { exact: false })
+    );
```

Comment:
The `waitForElement` wrapper is unnecessary here; you could just return `findByDisplayValue()` directly. 

(Also, the `waitForElement` method has been [deprecated in the latest versions](https://github.com/testing-library/dom-testing-library/issues/416).)

Reply:
Ah, I didn't know that, but that makes sense. Good to know RE: `waitForElement` too. Fixed in 1e3f86dd0a6e552896a87fe46106c37ffd697569